src/storage-blob-preview/azext_storage_blob_preview/_validators.py

```diff
@@ -1273,3 +1273,33 @@ def get_source_file_or_blob_service_client(cmd, namespace):
                                                       account_key=source_key,
                                                       share=source_share)
     ns['source_sas'] = source_sas
+
+
+def validate_text_configuration(cmd, ns):
+    DelimitedTextDialect = cmd.get_models('_models#DelimitedTextDialect', resource_type=CUSTOM_DATA_STORAGE_BLOB)
+    DelimitedJsonDialect = cmd.get_models('_models#DelimitedJsonDialect', resource_type=CUSTOM_DATA_STORAGE_BLOB)
+
+    if ns.input_format == 'csv':
+        ns.input_config = DelimitedTextDialect(
+            delimiter=ns.in_column_separator,
+            quotechar=ns.in_quote_char,
+            lineterminator=ns.in_record_separator,
+            escapechar=ns.in_escape_char,
+            has_header=ns.in_has_header)
+    if ns.input_format == 'json':
+        ns.input_config = DelimitedJsonDialect(delimiter=ns.in_line_separator)
+    if ns.input_format == 'parquet':
+        ns.input_config = "ParquetDialect"
```

Comment:
Better to use enum:`QuickQueryDialect.ParquetDialect` from SDK definition in case service team change the string later